### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by XinYenFon**
- HTML5 Valid bbc_tt
- Validation error on boards (<-p class="description_board"> missing when board description not showed inside the board)

Signed-off-by: Antes malinbay@gmail.com

### CODE REVIEW

File: Sources/Subs.php [hunk 1]
```diff
@@ -1509,8 +1509,8 @@ function parse_bbc($message, $smileys = true, $cache_id = '', $parse_tags = arra
 			),
 			array(
 				'tag' => 'tt',
-				'before' => '<tt class="bbc_tt">',
-				'after' => '</tt>',
+				'before' => '<div class="bbc_tt">',
+				'after' => '</div>',
```
Comment (mikemill) [inline Sources/Subs.php:1513]: You should give a CSS definition for bbc_tt that provides the same styling as the tt tag if you are going to change the tag.

**Comment by XinYenFon** [comment]
Yea that 's better my bad.

**Comment by emanuele45** [comment]
Another small issue: using a <div> for the [tt] tag will break the lines (divs are display:block by default), you should use a span. ;)

**Comment by XinYenFon** [comment]
I don't have my computer with me :/ Git wants to push every file to branch I'm going to close this PR and can you please take necessary fixes from here and push? thanks @emanuele45 :)

**Comment by mikemill** [comment]
See PR 190

**Pull request closed without merging**